securitylevel_test.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package eat
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSecLevel_Validate(t *testing.T) {
+	assert := assert.New(t)
+
+	var s1 SecurityLevel
```

Comment:
maybe better be explicit about the fact that we are checking `unrestricted`?

Reply:
The intent was to specifically check the "uninitialised" value here.

Reply:
the thing with go is that there is no such thing as uninitialised: this will get the default value for the underlying type which is 0, which also happens to be `SecLevelUnrestricted`

Reply:
Sure, hence my use of quotes. I guess the correct term would be the "default" value?  Point is I'm testing that, and it just happens to be `SecLevelUnrestricted`, rather then testing `unrestricted` specifically.

Reply:
cool, got it.